src/components/CurrentReviews.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+//not sure I need the spread operator actually
+function CurrentReviews({ reviews }, ...props) {
```

Comment:
not sure the spread operator helps here. you can either just use props object as a parameter or destructure it to extract all its values into own variables